receiver/swok8sobjectsreceiver/receiver.go

```diff
@@ -398,6 +399,20 @@ func getObjectHashes(udata *unstructured.Unstructured) (*objecthashes, error) {
 	udataCopy := udata.DeepCopy()
 	udataCopy.SetResourceVersion("") // do not report changes in resource version
 
+	udataBytes, err := udataCopy.MarshalJSON()
+	if err != nil {
+		return nil, err
+	}
+
+	timestampRegex := regexp.MustCompile(`/(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z))|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z))|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z))/`)
```

Comment:
What about a simpler regex, like `\b\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(\.\d+)?(Z|[+-]\d{2}:\d{2})\b`? While not as explicit, it's more readable and accepts also leap seconds and such.